staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder/builder.go

```diff
@@ -499,7 +499,9 @@ func addTypeMetaProperties(s *spec.Schema, v2 bool) {
 func (b *builder) buildListSchema(v2 bool) *spec.Schema {
 	name := definitionPrefix + util.ToRESTFriendlyName(fmt.Sprintf("%s/%s/%s", b.group, b.version, b.kind))
 	doc := fmt.Sprintf("List of %s. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md", b.plural)
-	s := new(spec.Schema).WithDescription(fmt.Sprintf("%s is a list of %s", b.listKind, b.kind)).
+	s := new(spec.Schema).
+		Typed("object", "").
```

Comment:
Do you have an example of the openapi this generates for a CRD?
We really lack golden data for this things ...

Reply:
https://github.com/kubernetes/kube-openapi/tree/master/pkg/schemaconv/testdata/crds/openapiv2

kube-openapi has a number of examples

Reply:
The new line though just adds "type": "object" to the list schema for CRDs

Reply:
> kube-openapi has a number of newly-added examples

Yes, but a change like this should almost certainly show into some golden file failing to diff somewhere. It would if this wasn't CRD related because we save the swagger.

Question, should we install a couple CRDs in the `hack/update-openapi-spec.sh` script to get them in the spec? We'd probably need to do that in addition to the existing spec, but I bet that'd be useful.

Reply:
https://github.com/kubernetes/kubernetes/issues/115351